pydarnio/borealis/borealis_formats.py

```diff
@@ -1751,14 +1748,359 @@ def array_dtypes(cls):
         In Borealis v0.5, blanked_samples was added as an array
         field.
         """
-        array_dtypes = super(BorealisRawrf, cls).array_dtypes()
+        array_dtypes = super(BorealisRawrfv0_4, cls).array_dtypes()
         array_dtypes.update({
             # Samples that occur during TR switching (transmission times)
             "blanked_samples": np.uint32
             })
         return array_dtypes
 
 
+# The following are the currently used classes, with additions according
+# to Borealis updates.
+
+class BorealisRawacf(BorealisRawacfv0_5):
+    """
+    Class containing Borealis Rawacf data fields and their types for the
+    current version of Borealis (v0.6).
+
+    See Also
+    --------
+    BaseFormat
+    BorealisRawacfv0_5
+    https://borealis.readthedocs.io/en/latest/borealis_data.html
+
+    Notes
+    -----
+    Rawacf data has been mixed, filtered, and decimated; beamformed and
+    combined into antenna arrays; then autocorrelated and correlated between
+    antenna arrays to produce matrices of num_ranges x num_lags.
+
+    See BaseFormat for description of classmethods and how they
+    are used to verify format files and restructure Borealis files to
+    array and site structure.
+
+    In v0.6, the following fields were added:
+    gps_locked, gps_to_system_time_diff, agc_status_word, and low_power_status_word.
+    array structured files contain the same fields, but with dims of [num_records].
+    """
+
+    @classmethod
+    def single_element_types(cls):
+        """
+        See BaseFormat class for description and use of this method.
+
+        Returns
+        -------
+        single_element_types
+            All the single-element fields in records of the
+            format, as a dictionary fieldname : type.
+
+        Notes
+        -----
+        In Borealis v0.6, gps_locked, gps_to_system_time_diff, agc_status_word,
+        and low_power_status_word were added as single element fields
+        """
+        single_element_types = super(BorealisRawacfv0_5,
```

Comment:
You want to call super on the class name itself - similar for other functions in this and the following classes

```suggestion
        single_element_types = super(BorealisRawacf,
```